Sources/SwiftDocCPluginUtilities/Snippets/SnippetExtractor.swift

```diff
@@ -65,6 +65,27 @@ public class SnippetExtractor {
     var _fileExists: (_ path: String) -> Bool = { path in
         return FileManager.default.fileExists(atPath: path)
     }
+
+    /// Returns all of the `.swift` files under a directory recursively.
+    ///
+    /// Provided for testing.
+    var _findSnippetFilesInDirectory: (_ directory: URL) -> [String] = { directory -> [String] in
+        guard let snippetEnumerator = FileManager.default.enumerator(at: directory,
+                                                                     includingPropertiesForKeys: [.isRegularFileKey],
+                                                                     options: [.skipsHiddenFiles]) else {
+            return []
+
+        }
+        var snippetInputFiles = [String]()
+        for case let potentialSnippetURL as URL in snippetEnumerator {
+            guard potentialSnippetURL.pathExtension == "swift" else {
```

Comment:
Should we lowercase the extension here?

```suggestion
            guard potentialSnippetURL.pathExtension.lowercased() == "swift" else {
```